### PR TITLE
Updated Readme

### PR DESCRIPTION
**Pull request by shfrz**
Fixes #

Changes made in this Pull Request:
 - 

PR Checklist
------------
 - [ ] Tests?
 - [x] Docs?
 - [ ] CHANGELOG updated?
 - [x] Issue raised/referenced?

### CODE REVIEW

**Comment by shfrz** [comment]
@MDAnalysis/gsoc-mentors could you please review my PR?

**Comment by shfrz** [comment]
@orbeckst  I have done some minor changes in the readme - fixed some typos and errors!
I

File: README.rst [hunk 1]
```diff
@@ -6,9 +6,13 @@
 
 |docs| |devdocs| |usergroup| |developergroup| |anaconda| |mybinder|
 
-MDAnalysis_ is a Python library for the analysis of computer simulations of many-body systems at the molecular scale, spanning use cases from interactions of drugs with proteins to novel materials. It is widely used in the scientific community and is written by scientists for scientists.
```
Comment (orbeckst) [inline README.rst]: We want to keep the sentence as is.

Calling the library "useful" is not helpful – if we didn't think it was useful we wouldn't bother working on it.

File: README.rst [hunk 2]
```diff
@@ -6,9 +6,13 @@
 
 |docs| |devdocs| |usergroup| |developergroup| |anaconda| |mybinder|
 
-MDAnalysis_ is a Python library for the analysis of computer simulations of many-body systems at the molecular scale, spanning use cases from interactions of drugs with proteins to novel materials. It is widely used in the scientific community and is written by scientists for scientists. 
+.. image:: https://pbs.twimg.com/profile_images/435418713366339584/ti3KF3aU_400x400.png
```
Comment (orbeckst) [inline README.rst]: The image shows up too big and generally distracts from the actual content of the README. Also, avoid linking to external content. There's a logo in the repo under docs/sphinx somewhere.

However, just drop the image inclusion to keep the README easily readable.

File: README.rst [hunk 3]
```diff
@@ -81,9 +85,9 @@ MDAnalysis issue tracker.)
 Guide for Developers
 ====================
 
-To setup a development environment and run the testsuite you can use this
+To set up a development environment and run the test suite you can use this
 guide_. If you are a new developer who would like to start contributing to
-MDAnalysis as a start you can increase our code coverage, the guides explain how
+MDAnalysis as a beginner, you can help increase our code coverage, the guides explain how
```
Comment (orbeckst) [inline README.rst]:
```suggestion
MDAnalysis, you can help increase our code coverage, the guides explain how
```

File: README.rst (hunk 2 above)
Comment (orbeckst) [reply README.rst]: As far as I can tell, you didn't address my comment. The diff still contains the changes that I wanted you to reverse. Just remove the image.

Comment (shfrz) [reply README.rst]: @orbeckst Did as directed!

**Comment by orbeckst** [comment]
Thank you!

**Pull request merged**